cmd/genesis.go

```diff
@@ -96,7 +80,135 @@ var verifyCmd = &cobra.Command{
 	},
 }
 
+var processCSVCmd = &cobra.Command{
+	Use: "csv",
+	Short: "Process a CSV file with account definitions " +
+		"(name, amount, key1, key2, key3, key4, key5, required_signatures)",
+	Args: cobra.MaximumNArgs(1),
+	Run: func(cmd *cobra.Command, args []string) {
+		csvFilePath, _ := cmd.Flags().GetString("file")
+		if csvFilePath == "" {
+			fmt.Println("CSV file path is required")
+			return
+		}
+
+		file, err := os.Open(csvFilePath)
+		if err != nil {
+			fmt.Printf("Failed to open CSV file: %v\n", err)
+			return
+		}
+		defer file.Close()
+
+		reader := csv.NewReader(file)
+		records, err := reader.ReadAll()
+		if err != nil {
+			fmt.Printf("Failed to read CSV file: %v\n", err)
+			return
+		}
+
+		// Process each record
+		for i, record := range records {
+			if i == 0 {
+				// Add new columns to the header
+				record = append(record, "vesting_address", "vault_address")
+				records[i] = record
+				continue
+			}
+
+			// Extract account details from the record
+			name := record[0]
+			amountStr := record[1]
+			keysStr := record[2:7]
+			requiredSignaturesStr := record[7]
+
+			// Convert amount and requiredSignatures to appropriate types
+			amount, err := strconv.ParseUint(amountStr, 10, 64)
+			if err != nil {
+				log.Fatalf("Error: amount %s for %s on line %d is invalid", amountStr, name, i)
+			}
+			amount *= constants.OneSmesh
+
+			requiredSignatures64, err := strconv.ParseUint(requiredSignaturesStr, 10, 8)
+			if err != nil {
+				log.Fatalf("Error: required signatures %s for %s on line %d is invalid", requiredSignaturesStr, name, i)
+			}
+			requiredSignatures := uint8(requiredSignatures64)
```

Comment:
Can we combine this with the `strconv` and just read the number directly into u8?